day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,148 @@
+frogs = 10
+  frogs.step(1, -1) do |x|
+   
+    if x == 2
+        puts "Two speckled frogs sat on a log 
+        eating some most delicious bugs.  
+        One jumped in the pool where its nice and cool,
+        then there was one speckled frog."
+    elsif x == 10
+        puts "Ten speckled frogs sat on a log 
+        eating some most delicious bugs.  
+        One jumped in the pool where its nice and cool,
+        then there were nine speckled frogs."
+    elsif x == 9
+        puts "Nine speckled frogs sat on a log 
+        eating some most delicious bugs.  
+        One jumped in the pool where its nice and cool,
+        then there were eight speckled frogs."
+    elsif x == 8
+        puts "Eight speckled frogs sat on a log 
+        eating some most delicious bugs.  
+        One jumped in the pool where its nice and cool,
+        then there were seven speckled frogs."
+    elsif x == 7
+        puts "Seven speckled frogs sat on a log 
+        eating some most delicious bugs.  
+        One jumped in the pool where its nice and cool,
+        then there were six speckled frogs."
+    elsif x == 6
+        puts "Six speckled frogs sat on a log 
+        eating some most delicious bugs.  
+        One jumped in the pool where its nice and cool,
+        then there were five speckled frogs."
+    elsif x == 5
+        puts "Five speckled frogs sat on a log 
+        eating some most delicious bugs.  
+        One jumped in the pool where its nice and cool,
+        then there were four speckled frogs."
+    elsif x == 4
+        puts "Four speckled frogs sat on a log 
+        eating some most delicious bugs.  
+        One jumped in the pool where its nice and cool,
+        then there were three speckled frogs."
+    elsif x == 3
+        puts "Three speckled frogs sat on a log 
+        eating some most delicious bugs.  
+        One jumped in the pool where its nice and cool,
+        then there were two speckled frogs."
+    else
+        puts "One speckled frog sat on a log 
+        eating some most delicious bugs.  
+        One jumped in the pool where its nice and cool,  
+        then there were no more speckled frogs!"
+    end
+end
+
+#And here is the far sexier code which I found while searching online for any built in methods that might be able to convert names to numbers. Apparently there are gems that do this, but likely they just look like the code below, just hidden back where they keep all the object and class rules and such. The cool part is it will go up as high as you like so long as you just continue to add suffixes into the has ex( 100 => "hundred" or 1000 => "thousand"
+def word_names(int)
+    numbers_to_name = {
+        10 => "ten",
+        9 => "nine",
+        8 => "eight",
+        7 => "seven",
+        6 => "six",
+        5 => "five",
+        4 => "four",
+        3 => "three",
+        2 => "two",
+        1 => "one"
+      }

```

Comment:
Yep, hashes are going to unlock a lot of solutions for you, good job researching!